client/.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
if we include .env then we don't need the stuff below this line, though I don't know which is better

@NLmeng might know though?

Reply:
technically, they serve different purposes, but we likely are only going to use .env, so we can either remove or keep it

Reply:
wait i just realized this isnt a filesystem LOL i totally read the . as /

if they're diff then ya whatever meng said